lib/needlessDisables.js

```diff
@@ -22,7 +21,7 @@ module.exports = function (results) {
 		const [enabled, options, stylelintResult] = settings;
 
 		/** @type {{[ruleName: string]: Array<DisabledRange>}} */
```

Comment:
[trivial] This `@type` annotation seems needless thanks to the type inference:

```suggestion
```

Reply:
@ybiquitous You're right, I think that type must have been needed when we were deep copying, but it is not important anymore.